ads/how-to-guides/override-layout.md

```diff
@@ -63,7 +64,7 @@ player.source = {
 ```
 
 :::note
-There is a limitation that double box (`DOUBLE`) is not supported on older smartTV's. If you want to play OptiView Ads on these devices, please override to another layout.
+There is a limitation that double box (`DOUBLE`) is not supported on Chromecast and older smartTV's. These revert the single layout automatically.
```

Comment:
@tjtheo Two things here:
1. "smartTV's" should be "smart TVs"
2. The last sentence should be "These revert to the single layout automatically."